### DISCUSSION TITLE
installing Centos 7.9 won't be executed on my mac m1 2020

### DISCUSSION THREAD

**Discussion by elhejj888** [utmapp/UTM#5826] [UTM Help (macOS)]
i tried to install Centos version 7 on my mac 2020 but whenever i try to launch the iso on UTM it stays stuck on a startup loop with three options as the screenshot shows ,
<img width="912" alt="Screenshot 2023-10-24 at 09 14 10" src="https://github.com/utmapp/UTM/assets/117082947/ddc0486a-a4f1-4c1b-ba50-8eaf79f17aa8">
whenever i select a value it launch a black window with a ticking  "_" for a second and then it returns to the first boot options , i tried multiple ISO but its always the same thing.

**Comment by estemendoza** [comment]
Having the same issue, tested with a MacBook Pro 2023, Sonoma 14.4, UTM 4.4.5

**Reply by elhejj888** [reply]
try new version it will work as Centos 9
